packages/synapse-react-client/src/components/RowDataTable/RowDataTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { SelectColumn } from '@sage-bionetworks/synapse-types'
+import { LabelLinkConfig } from '../CardContainerLogic'
+import { StyledTableContainer } from '../styled/StyledTableContainer'
+import { SynapseCardLabel } from '../GenericCard'
+
+export type RowDataTableProps = {
+  /** Array of string values corresponding to the data for a row */
+  rowData: string[]
```

Comment:
Allow passing `null` since Synapse Table row data may be null. If you need to do filtering, it's easy/centralized if you do it inside this component, then callers don't need to think about it/do it every time

```suggestion
  /** Array of string values corresponding to the data for a row */
  rowData: Array<string | null>
```